lessons/Data-Science/histograms-interpret/langs/en-us/index.adoc

```diff
@@ -0,0 +1,323 @@
+
+There is some setup required for the interactive activity that follows.
+
+- You will need flip chart paper, tape, and markers.
+- Divide your class into teams of three. All around your classroom, tape flip chart paper to the wall--one poster per team of three students.
+- On each piece of flip chart paper, write "Skew Left", "Skew Right", or "Symmetric", and then draw lines to divide the paper horizontally into three equal sections.
+}
+
+@lesson-instruction{
+- Your teacher has instructed you and your group to stand in front of a blank poster that says "Symmetric", "Skew Left", or "Skew Right" at the top.
+- The paper is divided into three horizontal sections. In the top section of your paper, draw a histogram that matches the shape assigned to you.
+}
+
+@teacher{Once all students have drawn their histograms, direct them to rotate to the next poster with a different shape.}
+
+@lesson-instruction{
+- You are now standing in front of a poster that identifies a shape and also includes a histogram of that shape.
+- In the second horizontal section of the poster, write a scenario or describe a dataset that would result in that distribution.
+- Leave the third section of the poster blank. We will use it another day!
+- Take a few minutes to walk around the classroom and read the scenarios your peers developed. Return to your seat when you have had a chance to see all of the posters.
+}
+
+@teacher{
+Spend some time debriefing with students, using the prompts below. Responses will vary for each group of students.
+}
+
+@QandA{
+@Q{Did all of the scenarios your classmates developed correctly represent the given shape?}
+
+@Q{Did you notice any especially creative scenarios? Any surprising scenarios?}
+
+@Q{Were there any scenarios that were represented more than once? Why do you think those scenarios were popular?}
+}
+
+@teacher{
+The third section of the posters will remain blank today. Students will complete this section in the future when they learn about @lesson-link{box-plots}.
+}
+
+
+@lesson-instruction{
+Try it on your own! Complete @printable-exercise{reading-histograms-matching.adoc}, where you will match descriptions of video ratings to histograms that could fit the data.
+}
+
+@teacher{Go over correct responses as a class.}
+
+
+=== Synthesize
+
+@QandA{
+@Q{What strategies did your group use in brainstorming scenarios to match histogram shape?}
+@Q{For which distributions was it easiest to come up with an example?}
+@Q{For which distributions was it hardest to come up with an example?}
+}
+
+
+== Histograms and Measures of Center
+
+@objective{histograms-and-measures-of-center}
+
+=== Overview
+
+Students combine apply their knowledge histograms, measures of center, and shape.
+
+=== Launch
+
+Can you compute and interpret different measures of center from a histogram? Let's try it.
+
+@QandA{
+The histogram below shows the number of children per home in the Broadmoor neighborhood.
+
+@center{@image{images/number-children.png, 250}}
```

Comment:
I think we need a different histogram here with a bin size that's bigger than one. 
Strategy 1 below won't work for the histograms on the page they will be doing independently or most histograms they will encounter...

Assuming there are also some histograms where we can identify which bin the median lands in and some histograms where we can't, although I haven't tried to generate an example for the latter yet and I could be wrong.

@schanzer might also have ideas here?

Reply:
This problem comes directly from the research. I need to add a citation. I'm happy to change it if you'd like, but I think the authors describe its purpose pretty clearly? LMK if you disagree.

![image](https://github.com/user-attachments/assets/e38b8d3c-9897-4b40-8e8d-adf220a2da79)

Reply:
maybe I need to explain the purpose more to teachers? LMK what you're thinking.

Reply:
@retabak what I'm thinking is that the researchers used both kinds of histograms.
the latter is the one that students are generally being asked to reason about (including elsewhere in this lesson)
but none of the latter made it into the body of the lesson plan
and that doesn't make sense.

I'm not attached to what happens with the former. if we're introducing it in the lesson plan then we should probably also offer some practice with one like it on a page that will be in the workbook.

Reply:
well, the former is a scaffold. So I think it belongs in the lesson plan, and is appropriate as a warm up. I think it helps students realize that they CAN reason about measures of center by looking at the information provided via histograms. I don't think it requires more practice on a workbook page, because (as you said) the latter is the one that students are being asked to reason about. 

if you believe there should be more practice with it, i think an additional workbook page would be necessary (there's no room to squeeze in another problem on the existing worksheet). LMK if that's what you want, still.

Reply:
please add the other kind of histogram to the body of the lesson plan.